### PR TITLE
Truncat message to 100 before logging/uploading it

### PR DESCRIPTION
**Pull request by abenbachir**

### CODE REVIEW

File: OmsAgent/watcherutil.py [hunk 1]
```diff
@@ -223,15 +223,23 @@ def upload_telemetry(self):
                             message = status_data["message"]
 
                             event = self.create_telemetry_event(operation,operation_success,message,"300000")
-                            self._hutil_log("Writing telemetry event: "+event)
                             self.write_waagent_event(event)
```
Comment (ers-msft) [inline OmsAgent/watcherutil.py]: should we be truncating the message sent to waagent as well?  would like to ensure that is not overwhelmed as well. thanks

Comment (abenbachir) [reply OmsAgent/watcherutil.py]: Yes, we could, is it ok to do that ?

Comment (simathih) [reply OmsAgent/watcherutil.py]: yeah, we saw that it grew to 600MB for the customer yesterday

Comment (ers-msft) [reply OmsAgent/watcherutil.py]: yes for now please also truncate waagent logging, we want to ensure we do not overload that either.

Comment (abenbachir) [reply OmsAgent/watcherutil.py]: Done, we are now truncating it to 100 for waagent too

**Pull request merged**